src/AppBundle/DataFixtures/TeamFixtures.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace AppBundle\DataFixtures;
+
+use AppBundle\Entity\Client;
+use AppBundle\Entity\Team;
+use AppBundle\Entity\User;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class TeamFixtures extends AbstractDataFixture implements OrderedFixtureInterface
+{
+    public function doLoad(ObjectManager $manager)
+    {
+        // Create teams
+        $this->createTeam('PA', 'Public Authority', $manager);
+        $this->createTeam('PROF', 'Professional', $manager);
+
+        $manager->flush();
+    }
+
+    private function createTeam($code, $name, $manager) {
+        // Create team
+        $team = new Team($name . ' Team');
+        $manager->persist($team);
+
+        // Create team admin user
```

Comment:
Whenever a one line comment can nicely describe the next few lines, it's usually a candidate to create a method with the same name as that one line. I'm a big fan of writing descriptive code rather than untrustworthy comments. Not a blocker as this is just fixture code, just a comment.

Reply:
Yeah, that's definitely good advice. I'll bear it in mind for the future, but I don't think it'll make a great difference here because the duplication is quite minimal.